client/ayon_houdini/plugins/publish/validate_subset_name.py

```diff
@@ -21,9 +21,7 @@ class FixProductNameAction(RepairAction):
 
 class ValidateSubsetName(plugin.HoudiniInstancePlugin,
```

Comment:
Technically this class name should be updated to, but:

- Existing "optional" toggles on existing publish instances would not follow along - I don't believe we have any conversion mechanism for this?
- Settings would need to be changed to (and get 'conversion' implemented for it)

Reply:
> Existing "optional" toggles on existing publish instances would not follow along - I don't believe we have any conversion mechanism for this?

You can implement classmethod `convert_attribute_values` which can do that, object of `PublishAttributes` is passed in.